### PR TITLE
docs: added a link to the contributing guidelines in the PR template

### LINKED ISSUE #1807: Update contributing guidelines
## Description

Update our contributing guidelines to indicate that an issue needs to be created first before creating a PR. The issue will allow us to track the work and prioritize it.

### PR DESCRIPTION
**Pull request by nickytonline**
### Summary

Updates our contributing guidelines to indicate that an issue must first be opened and discussed before submitting a PR.

### Test plan

N/A. This only updates documentation.

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

Closes #1807

### Standard checks:

- [ ] Check the Deploy Preview's Demo site for your PR's functionality
- [ ] Add docs when necessary

---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.

An alpacca with sunglasses on

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 🎉 Thanks for considering contributing to this project! 🎉
 
-These guidelines will help you send a pull request.
+When contributing to this repository, please first discuss the change you wish to make via an
```
Comment (nickytonline) [inline CONTRIBUTING.md:5]: Feel free to suggest better wording.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,9 +40,9 @@ We use [Conventional Commit messages](https://www.conventionalcommits.org/) to a
 
 Most common commit message prefixes are:
 
-* `fix:` which represents bug fixes, and generate a patch release.
-* `feat:` which represents a new feature, and generate a minor release.
-* `feat!:`, `fix!:` or `refactor!:` and generate a major release.
+- `fix:` which represents bug fixes, and generate a patch release.
```
Comment (nickytonline) [inline CONTRIBUTING.md:43]: Looks like prettier in action.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,12 +2,15 @@
...
+help you and to help the maintainers gather information.
 
-If you're submitting an issue instead, please skip this document.
+Before working on an issue, ask to be assigned to it. This makes it clear to other potential contributors that someone
+is working on the issue.
 
-If your pull request is related to a typo or the documentation being unclear, please click on the relevant page's `Edit`
-button (pencil icon) and directly suggest a correction instead.
+When creating a PR, please fill out the template to the best of your abilities. Do not delete it. The information in the
+template helps maintainers review your pull request.
```
Comment (orinokai) [inline CONTRIBUTING.md:13]: [sand] not a big deal, but maybe this sounds a bit heavy handed?
```suggestion
When creating a PR, please use the template. The information in the template helps maintainers review your pull request.```

**Review by orinokai — APPROVED** [review]
Thanks @nickytonline! One suggestion, but take it or leave it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (sarahetter) [reply CONTRIBUTING.md:13]: +1

**Review by sarahetter — APPROVED** [review]

**Pull request merged**